src/Website/Models/Account/EditLogin.cs

```diff
@@ -1,19 +1,28 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.Web.Mvc;
 
-namespace mbsoft.BrewClub.Models
+namespace mbsoft.BrewClub.Website.Models.Account
 {
-	public class LoginModel
+	public class EditLogin
```

Comment:
"Edit" might be a bit of a stretch, but I think the ideal is to have models be either Edit/View/Create, so Edit makes the most sense here.